src/sage/rings/function_field/order.py

```diff
@@ -1773,7 +1773,7 @@ def decomposition(self, ideal):
         finite-dimensional algebra, a construct for which we do
         support primary decomposition.
 
-        See :trac:`attachment/ticket/28094/decomposition.pdf`
+        See :trac:`28094` and https://github.com/sagemath/sage-prod/files/10659303/decomposition.pdf.gz
```

Comment:
You can drop the `-prod` in the URL.

Reply:
Will do.  I'll note that the attachment links right now on issues like #28094 point to the `sage-prod` version of the url.